### PR TITLE
unify dependencies versions

### PR DESCRIPTION
**Pull request by harksin**
see: 
* https://doc.rust-lang.org/nightly/cargo/reference/specifying-dependencies.html#inheriting-a-dependency-from-a-workspace
* https://doc.rust-lang.org/nightly/cargo/reference/workspaces.html#the-dependencies-table

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -14,3 +14,15 @@ members = [
...
 default-members = ["aster"]
+
+
+[workspace.dependencies]  
+tokio = "1.28.2"
+anyhow = "1.0.71"
+async-trait = "0.1.68"
+chrono = "0.4.26"
+log = "0.4.18"
+crossbeam-channel = "0.5.8"
```
Comment (lucido-simon) [inline Cargo.toml:25]: We should be wary of which of #100 and this PR merges first, since this dep changes.

File: common/Cargo.toml [hunk 2]
```diff
@@ -6,9 +6,9 @@ edition = "2021"
...
 
 [dependencies]
-async-trait = "0.1.68"
-crossbeam-channel = "0.5.1"
-thiserror = "1.0.40"
-chrono = "0.4.26"
-anyhow = "1.0.71"
-sqlx = { version = "0.6.3", features = [ "runtime-tokio-native-tls" ] }
+async-trait = { workspace = true }
+crossbeam-channel = { workspace = true }
```
Comment (lucido-simon) [inline common/Cargo.toml:10]: same here

File: connector/Cargo.toml [hunk 3]
```diff
@@ -7,10 +7,10 @@ edition = "2021"
...
-anyhow = "1.0.71"
-thiserror = "1.0.40"
-crossbeam-channel = "0.5.1"
-chrono = "0.4.26"
-log = "0.4.18"
+async-trait = { workspace = true }
+sqlx = { workspace = true , features = [ "runtime-tokio-native-tls", "postgres", "chrono" ] }
+anyhow = { workspace = true }
+thiserror = { workspace = true }
+crossbeam-channel = { workspace = true }
```
Comment (lucido-simon) [inline connector/Cargo.toml:14]: same here

**Comment by lucido-simon** [comment]
LGTM but as stated in the comment, mayble merge #100 first and change the deps in this PR?

File: Cargo.toml (hunk 1 above)
Comment (harksin) [inline Cargo.toml:25]:
```suggestion
```

File: connector/Cargo.toml (hunk 3 above)
Comment (harksin) [inline connector/Cargo.toml:14]:
```suggestion
```

**Pull request merged**